vector-search-data-types.md

```diff
@@ -126,7 +129,7 @@ When comparing vector constants, consider performing an [explicit cast](#cast) f
 
 ## Arithmetic
 
-Vector data types support element-wise arithmetic operations `+` (addition) and `-` (subtraction). However, performing arithmetic operations between vectors with different dimensions results in an error.
+Vector data types support arithmetic operations `+` (addition) and `-` (subtraction). However, arithmetic operations between vectors with different dimensions are not supported and will result in an error.
 
 Examples:
 
```

Comment:
L145:
```
[tidb]> SELECT VEC_FROM_TEXT('[2,3,4]') - VEC_FROM_TEXT('[1,2,3]');
```

Reply:
fixed in [`cb7e4dc` (#18779)](https://github.com/pingcap/docs/pull/18779/commits/cb7e4dc1319617b1d525d083235bdd53d4099a08)